### ISSUE TITLE
Message parsing is not working as expected.

### ISSUE THREAD

**Issue by r0hi7** [r0hi7/Trashemail#25]
**Describe the bug**
Message parsing is not working as expected. If you send a very basic HTML mail from any mail service. Following parameters a coming out to be true.
```
MailParser    : Is plain content true
MailParser    : Is HTML content true
MailParser    : Is multipart true
MailParser    : Is plain content Empty false
```
Due to which, this statement is never reached. https://github.com/r0hi7/Trashemail/blob/b67de8bfface2386071b32b9a9930d90a77d8b20/src/main/java/io/github/trashemail/utils/MailParser.java#L39
Because this is evaluated to **false**: https://github.com/r0hi7/Trashemail/blob/b67de8bfface2386071b32b9a9930d90a77d8b20/src/main/java/io/github/trashemail/utils/MailParser.java#L38

**To Reproduce**
1. Send an HTML mail from good with the following content.
```Markdown
**Bold text**
_Italic_
```

**Expected behavior**
Since this is multipart mail, the HTML field should be processed.  And HTML message should be recorded.

**Comment by r0hi7** [comment]
@nishitm, I am checking on this. But since, you merged the PR for this, can you cross verify this issue?

**Comment by r0hi7** [comment]
The RC of this bug was that it was not ever checking for multipart data.

**Comment by r0hi7** [comment]
Fixed in #26
